testgrid/web/src/components/views/InstanceTable.jsx

```diff
@@ -216,7 +226,15 @@ export default class InstanceTable extends React.Component {
     const rows = Object.keys(this.props.instancesMap).map(kurlURL => {
       return (
         <tr key={kurlURL}>
-          <td><span className="url" onClick={() => this.viewInstanceInstaller(this.props.instancesMap[kurlURL][0])}>{kurlURL}</span></td>
+          <td>
+            <span className="url" onClick={() => this.viewInstanceInstaller(this.props.instancesMap[kurlURL][0])}>{kurlURL}</span>
+            {(this.props.instancesMap[kurlURL][0].upgradeURL && this.props.instancesMap[kurlURL][0].upgradeURL !== "") &&
```

Comment:
im on js expert but i think this is redundant

Reply:
true, was going to comment on that, but it doesn't hurt